src/Console/HelperRegistry.php

```diff
@@ -55,8 +55,7 @@ public function setIo(ConsoleIo $io): void
      * Part of the template method for Cake\Core\ObjectRegistry::load()
      *
      * @param string $class Partial classname to resolve.
-     * @return string|null Either the correct class name or null.
-     * @psalm-return class-string
+     * @return class-string|null Either the correct class name or null.
```

Comment:
TypeResolver doesn't seem to handle `class-string` yet. Maybe I can add it in a PR.